pkg/cloud/services/cloudformation/bootstrap.go

```diff
@@ -191,6 +191,8 @@ func controllersPolicy(accountID, partition string) *iam.PolicyDocument {
 					"ec2:RevokeSecurityGroupIngress",
 					"ec2:RunInstances",
 					"ec2:TerminateInstances",
+					"tag:GetResources",
+					"tag:TagResources",
```

Comment:
I think we can remove this permission - we do not currently use it as far as I'm aware.

Reply:
> I think we can remove this permission - we do not currently use it as far as I'm aware.

this only uses the get , I will remove the set later.